[CI][dashboard] Rename cpu-x86 to cpu_x86

desertfire opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #131658

Summary: '-' is used as a special separator by upload_dynamo_perf_stats.py, so switch to '_' instead.

desertfire committed:
[CI][dashboard] Rename cpu-x86 to cpu_x86

Summary: '-' is used as a special separator by upload_dynamo_perf_stats.py, so switch to '_' instead.

[ghstack-poisoned]

desertfire commented:
Testing in https://github.com/pytorch/pytorch/actions/runs/10079358647/workflow

huydhn commented:
> @huydhn , [pytorch/test-infra#5491](https://github.com/pytorch/test-infra/pull/5491) needs to be updated correspondingly.

Got it!  That should be an easy tweak on that PR.

desertfire committed:
Update on "[CI][dashboard] Rename cpu-x86 to cpu_x86"


Summary: '-' is used as a special separator by upload_dynamo_perf_stats.py, so switch to '_' instead.

[ghstack-poisoned]

desertfire commented:
@pytorchbot merge -f "only affects inductor-perf-test-nightly-x86.yml"
